src/components/layout/Box/Box.scss

```diff
@@ -0,0 +1,23 @@
+@use '../variables.scss' as v;
+
+#{v.$boxBlock} {
+    box-sizing: border-box;
+
+    &_overflow_hidden {
+        overflow: hidden;
+    }
+
+    &_overflow_auto {
+        overflow: auto;
+    }
+
+    &_overflow_x {
+        overflow-x: hidden;
+        overflow-y: auto;
+    }
```

Comment:
Is it a typo? `overflow_x` mod for `overflow-y: auto`

Reply:
no, `overflow_x` means that only for `x` axis content is hidden